### ISSUE TITLE
Web serial is available in browser now

### ISSUE THREAD

**Issue by james-portman** [speeduino/Ardu-Stim#52]
Bar the avrdude/direct binaries, the rest of this can now be done in browser only (without node).

I can't get the node stuff to work in Linux which is a shame.
Should be possible to do almost all of this in plain JS/HTML now using web serial

https://codelabs.developers.google.com/codelabs/web-serial#7

**Comment by james-portman** [comment]
OK, my point still stands.
I cannot get it to build as-is on my computer.
It could just be straight javascript and work on Windows/Mac/Linux all in the same way without external things like node, electron

**Comment by karniv00l** [comment]
Yeah, it could be nice, but the support is pretty bad right now: https://caniuse.com/web-serial
